CHANGES.md

```diff
@@ -1,3 +1,27 @@
+## edge-19.6.2
+
+* CLI
+  * Added the `--linkerd-cni-enabled` flag to the `install` subcommands so that
+    `NET_ADMIN` capability is omitted from the control plane's PSP
+* Controller
+  * Fixed security context values so that injection does not fail on
+    restricted PSPs (thanks @codeman9!)
+  * Fixed webhook failure policy in order to avoid timing issues in HA mode
```

Comment:
The reason for that change is to avoid un-injected applications from being deployed, if the proxy injector fails as a result of unexpected errors (like network timeout etc.). Something like:

_Default the webhook failure policy to "fail", to account for unexpected errors during auto-proxy injection. This ensures un-injected applications aren't deployed._